reference/carts.v3.yml

```diff
@@ -575,7 +577,6 @@ paths:
                             coupons: []
                             discount_amount: 0
                             coupon_amount: 0
-                            original_price: 27.95
```

Comment:
@bc-traciporter is this right?

Reply:
@slsriehl yes this is correct. `original_price` appears in the response.

Reply:
@bc-traciporter this is part of the error you found. original_price should appear and does appear in the response, but this PR has it removed. thanks for clarifying.